docs/release-notes/workload-alerting.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**New Features**
+
+-  Webhooks: Add support for monitoring experiment by workspaces, editing webhook URL and triggering
+   webhook from code. For more details, refer to :ref:`supported-webhook-triggers`.
```

Comment:
:orphan:

**New Features**

-  Webhooks: Add support for experiment monitoring and alerting. Capabilities include creation of workspace-level subscriptions with "open" and "run" specific modes, CUSTOM trigger including custom triggers for code-based alerts, webhook exclusion for specific experiments, and editable webhook URLs. For details, visit :ref:`supported-webhook-triggers`.